apps/playground/src/app/test/test.component.ts

```diff
@@ -74,4 +74,14 @@ export class TestComponent {
   public goBack() {
     this.location.back();
   }
+
+  public dashboardFormatters() {
+    // const cf          = new google.visualization.ColorFormat ();
```

Comment:
Please remove the comments, I guess they are not needed.

Reply:
Done, and done.